docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -612,6 +612,11 @@ iceberg.snowflake-catalog.password=secret
 iceberg.snowflake-catalog.database=db
 ```
 
+The Snowflake catalog has a following limitations in Trino due to the [limitation of Iceberg Snowflake catalog implementation](https://docs.snowflake.com/en/user-guide/tables-iceberg-catalog#limitations):
+- The Snowflake catalog is a read-only catalog.
+- Does not support partitioned Iceberg tables.
```

Comment:
This sentence is confusing for me. Does it mean the catalog doesn't support "reading" partitioned tables?

Reply:
We have to clarify all this more .. for example also .. what does "read-only" mean? Cant create a table? alter a table? Or is this all covered in the linked docs . then we can omit the whole list. I will review tomorrow.

Reply:
sorry I meant snowflake Iceberg tables do not support partitions and therefor the snowflake Iceberg catalog.

---
clarified that it supports only SELECT statement as documented in the snowflake iceberg catalog link.